internal/npm_install/test/golden/unidiff/BUILD.bazel.golden

```diff
@@ -40,7 +47,8 @@ js_library(
 )
 js_library(
     name = "unidiff__contents",
-    srcs = [":unidiff__files", ":unidiff__nested_node_modules"],
+    srcs = [":unidiff__nonambient_files", ":unidiff__nested_node_modules"],
+    ambient_srcs = [":unidiff__ambient_files"],
```

Comment:
this partly answers my question about why `js_library` has to change, but I wonder if these attributes are only used privately or if it's something users will need to use in some case